### PR TITLE
fixes(#10): replace test-logger implementation

### PR DESCRIPTION
**Pull request by abacaphiliac**
per https://packagist.org/packages/gamez/psr-testlogger

### CODE REVIEW

**Comment by abacaphiliac** [comment]
@holtkamp was this what you had in mind? unfortunately, the PSR TestLogger is only available since `psr/log:^1.1`.

File: test/PsrSqlLoggerIntegrationTest.php [hunk 1]
```diff
@@ -40,41 +39,45 @@ protected function setUp()
 
     /**
      * @param integer $index
-     * @return Record
+     * @return \stdClass
      */
     private function getRecordByIndex($index)
```
Comment (holtkamp) [inline test/PsrSqlLoggerIntegrationTest.php]: Maybe instead of using a `@return \stdClass`, use a return type `: \stdClass`?

**Review by holtkamp — CHANGES_REQUESTED** [review]
Yeah indeed, I would suggest to use return types where possible, for example:

```php
private function getRecordByIndex($index) : \stdClass
```

**Comment by abacaphiliac** [comment]
@holtkamp ah, of course! carry-over from `php:<7` : (

i have made the changes!

**Review by holtkamp — APPROVED** [review]

**Pull request merged**